_codux/boards/card-collection-2.board.css

```diff
@@ -0,0 +1,89 @@
+.card-collection {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    font-family: Arial, sans-serif;
+    min-height: 800px;
+    max-height: 700px;
+
+    @media (width<= 1024px) {
```

Comment:
Is this supposed to be `width < 1024px`

See the recording

https://github.com/user-attachments/assets/8115cb32-20da-415a-bfcf-a67081bc34c9